Issue#31 functionality in search walker

Darkboy5120 opened the pull request:
# Description
the focus in this pull was introduce the first version of walkers and clients positions match, the algoritm check boths users's distance to show to the client nears walkers, so boths users are updating theirs position to server in order to find near users.

Darkboy5120 committed:
refactorizado la actualizacion de positiones de usuarios

- antes la informacion usada por la api de realtime database no me dejaba ver la informacion en la interface de firebase, ahora use firestore database y solo agrege un campo lastPosition a los usuarios en Users

Darkboy5120 committed:
lista de walkers cercanos

- aun falta evaular las distancia entre el client y el walker

Darkboy5120 committed:
validacion de distancia

Darkboy5120 committed:
codeclimate fixes

- to many returns in function
- duplication

Darkboy5120 committed:
codeclimate duplication fix

YilKanda commented on src/assets/controllers/firebaseShortcuts.js:
```diff
@@ -0,0 +1,45 @@
+
+const handleThen = (req, callback) => {
+  return req.catch(e => {
+    console.error(e);
```
Don't put console.log(), when you push your code from GitHub. Instead you can make use of Alert.alert()

Darkboy5120 replied:
done

YilKanda commented on src/assets/controllers/firebaseShortcuts.js:
```diff
@@ -0,0 +1,45 @@
+  getCollection,
+};
+
+export default fbShortcuts;
```
Add a blank line

Darkboy5120 replied:
done

YilKanda commented on src/assets/controllers/updateRealtimeLocation.js:
```diff
@@ -71,6 +68,7 @@ const listen = async () => {
 
 const realtimeLocation = {
   listen: listen,
+  // listen: () => {},
```
Don't add code in comments on GitHub.

Darkboy5120 replied:
i forgot remove it

YilKanda commented on src/components/LoadingSpinner/index.js:
```diff
@@ -0,0 +1,33 @@
+  );
+};
+
+export default LoadingSpinner;
```
Add a blank line

Darkboy5120 replied:
done

Darkboy5120 committed:
blank space

Darkboy5120 committed:
replace it console with alert

Darkboy5120 committed:
remove it comment

Darkboy5120 committed:
added blank space

YilKanda reviewed (approved):
I recommend that when you make fixes in the code, instead of creating a commit for each fix, you can use the Git amend:

```
git add .
git commit --amend --no-edit
git push --force-with-lease origin your-branch
```

Darkboy5120 committed:
fixed error while trying to access lastPosition

Darkboy5120 committed:
removed useless line

Darkboy5120 committed:
installed redux to handle user's sesions data

- to handle lastPosition and user uid between screens easily

Darkboy5120 committed:
lastPosition updated by redux

- now redux save the last position and bring the info to clientWalkers' screen in order to find near walkers

Darkboy5120 committed:
auto sign fixed

- now the auto sign checks the user's type in order to open the respective interface

Darkboy5120 committed:
codeclimate duplication fix

- new modules created to import all navigation screens

Darkboy5120 committed:
codeclimate fix
